src/styles/components/badges/styles.less

```diff
@@ -16,163 +16,167 @@ components/badges.less
 /* -----------------------------------------------------------------------------
 --------------------------------------------------------------------------------
 
-Badge
+Badges
 
 --------------------------------------------------------------------------------
 ----------------------------------------------------------------------------- */
 
-.badge {
+& when (@badges-enabled) {
 
-  color: @body-background-color;
-  font-size: @body-text-font-size;
-  line-height: @body-text-line-height;
-  padding: 0 @base-spacing-unit * 0.25;
-  background-color: @button-default-active-background-color;
-  border-radius: 1000px !important;
-  min-width: @base-spacing-unit;
+  .badge {
 
-  &:empty {
+    color: @body-background-color;
+    font-size: @body-text-font-size;
+    line-height: @body-text-line-height;
+    padding: 0 @base-spacing-unit * 0.25;
+    background-color: @button-default-active-background-color;
+    border-radius: 1000px !important;
+    min-width: @base-spacing-unit;
 
-    display: none;
+    &:empty {
 
-  }
+      display: none;
+
+    }
+
+    &.inverse {
 
-  &.inverse {
+      background-color: rgba(255, 255, 255, 0.05);
 
-    background-color: rgba(255, 255, 255, 0.05);
+    }
 
   }
 
-}
 
 
+  .active {
 
-.active {
+    .badge {
```

Comment:
Interesting way to apply an active state.  Seems like this could create conflicts in certain scenarios.  I guess it works for now, but might be worth changing in future updates to be more badge specific and less tied to it's parent component.